polaris-react/src/components/TopBar/components/UserMenu/UserMenu.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 
-import type {IconableAction} from '../../../../types';
 import {Avatar} from '../../../Avatar';
 import type {AvatarProps} from '../../../Avatar';
 import {MessageIndicator} from '../../../MessageIndicator';
 import {Menu} from '../Menu';
 import type {MenuProps} from '../Menu';
 import {Text} from '../../../Text';
+import type {ActionListProps} from '../../../ActionList';
 
 import styles from './UserMenu.scss';
 
 export interface UserMenuProps {
   /** An array of action objects that are rendered inside of a popover triggered by this menu */
-  actions: {items: IconableAction[]}[];
+  actions: ActionListProps['sections'];
```

Comment:
Are we sure this isn't a breaking change?

Reply:
This prop is passed directly into the [Menu component here](https://github.com/Shopify/polaris/blob/7a6a91dce2d41110e439ba889143becc4b17e3c8/polaris-react/src/components/TopBar/components/Menu/Menu.tsx#L15), which takes the same type. I think having `{items: IconableAction[]}[]` made sense before because `UserMenu` didn't need some of the props like `title` etc.